### PR TITLE
Update secure-data.md

### LINKED ISSUE #34264: Use git sparse-checkout
### Description

🚨 Looks like you arrived from the internal review site 
🚨 Do not enter Microsoft confidential information here

[Enter feedback here]

### Page URL

https://review.learn.microsoft.com/en-us/aspnet/core/security/authorization/secure-data?view=aspnetcore-9.0&branch=pr-en-us-34239

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/secure-data.md

### Document ID

7c151ed5-fd78-84fd-a08a-88912990c4ca

### Article author

@Rick-Anderson

### PR DESCRIPTION
**Pull request by danuw**
Adding a tip to ensure the visitor of trying to run through this guide can git clone only the sub folders needed for this tutorial from the docs for an even more lightweight and faster experience (For context, I was trying to do this on the train and internet was patchy and this simply helped get me progressing as I could not get the full repo downloaded otherwise).

I used this to ensure I was using the right syntax https://learn.microsoft.com/en-us/contribute/content/markdown-reference#alerts-note-tip-important-caution-warning

Fixes #34264

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/authorization/secure-data.md](https://github.com/dotnet/AspNetCore.Docs/blob/578e4b650346c61a3f9fba80ac0dde787c85fc3b/aspnetcore/security/authorization/secure-data.md) | [aspnetcore/security/authorization/secure-data](https://review.learn.microsoft.com/en-us/aspnet/core/security/authorization/secure-data?branch=pr-en-us-34239) |

### CODE REVIEW

File: aspnetcore/security/authorization/secure-data.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ This tutorial is advanced. You should be familiar with:
 
 [Download](xref:index#how-to-download-a-sample) the [completed](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/secure-data/samples) app. [Test](#test-the-completed-app) the completed app so you become familiar with its security features.
 
+> [!TIP]
+> To only clone the subfolder, use [`git sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) command as follows:
+> git clone --depth 1 --filter=blob:none https://github.com/dotnet/AspNetCore.Docs.git --sparse
+> cd AspNetCore.Docs
+> git sparse-checkout init --cone
+> git sparse-checkout set aspnetcore/security/authorization/secure-data/samples
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:75]: >

Comment (danuw) [reply aspnetcore/security/authorization/secure-data.md:75]: Hi  @Rick-Anderson was there a comment to resolve please?

Comment (Rick-Anderson) [reply aspnetcore/security/authorization/secure-data.md]: Suggestions don't take ``` so I changed it in a commit

**Comment by danuw** [comment]
Thank you for the update. "download" shows with an extra 'w' btw. Sorry about that

File: aspnetcore/security/authorization/secure-data.md [hunk 2]
```diff
@@ -67,6 +67,16 @@ This tutorial is advanced. You should be familiar with:
 
 [Download](xref:index#how-to-download-a-sample) the [completed](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/secure-data/samples) app. [Test](#test-the-completed-app) the completed app so you become familiar with its security features.
 
+> [!TIP]
+> Use [`git sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) to only downlowd the sample subfolder
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:71]:
```suggestion
> Use [`git sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) to only download the sample subfolder
```

**Comment by Rick-Anderson** [comment]
@danuw  that's super helpful, thanks so much.

**Comment by danuw** [comment]
My pleasure

**Pull request merged**